src/utils/tests/requestHelper.js

```diff
@@ -6,20 +6,49 @@ class RequestHelper {
     this.request = request(application);
   }
 
-  post(url, code, obj) {
-    return this.request
+  post(url, code, obj, token) {
+    const setting = this.request
       .post(url)
       .type('form')
       .send(obj)
-      .set('Accept', /application\/json/)
-      .expect(code);
+      .set('Accept', /application\/json/);
+    if (token) {
+      setting.set('x-access-token', token || null);
+    }
+
+    return setting.expect(code);
+  }
+
+  put(url, code, obj, token) {
+    const setting = this.request
+      .put(url)
+      .type('form')
+      .send(obj)
+      .set('Accept', /application\/json/);
+    if (token) {
+      setting.set('x-access-token', token || null);
+    }
+    return setting.expect(code);
+  }
+
+  delete(url, code, obj, token) {
```

Comment:
you can make a generic factory that will take `method` param and do
```
this.request[method].type(....)
```

and then just do 
`get = (...params) => factory('get', ...params)`;